File: javer/src/main/java/com/uuhnaut69/javer/service/DishService.java
package com.uuhnaut69.javer.service;

import com.uuhnaut69.javer.model.Dish;
import com.uuhnaut69.javer.resource.dto.DishRequest;

import java.util.List;
import java.util.UUID;

public interface DishService {

  List<Dish> findAllByRestaurantId(UUID restaurantId);

  Dish create(UUID restaurantId, DishRequest dishRequest);
}

File: api/kover/src/main/kotlin/com/uuhnaut69/kover/service/DishService.kt
package com.uuhnaut69.kover.service

import com.uuhnaut69.kover.model.Dish
import com.uuhnaut69.kover.resource.dto.DishRequest
import java.util.*

interface DishService {

    fun findAllByRestaurantId(restaurantId: UUID): List<Dish>

    fun create(restaurantId: UUID, dishRequest: DishRequest): Dish
}